src/excmds.ts

```diff
@@ -3531,7 +3531,7 @@ export function autocontain(...args: string[]) {
     let [pattern, container] = args
 
     if (!urlMode) {
-        pattern = `^https?://[^/]*${pattern}/`
+        pattern = `^https?://([^/]*\\.|)${pattern}/`
```

Comment:
```suggestion
        pattern = `^https?://([^/]*\.|)${pattern}/`
```
It's either wrong here and in config.ts or wrong in the docstring :)

Reply:
Two backslashes are needed in JavaScript strings. I do not know if the tridactyl config parser escapes/unescapes content.

```
Welcome to Node.js v14.5.0.
Type ".help" for more information.
> new RegExp('\.').test('a')
true
> new RegExp('\\.').test('a')
false
> new RegExp('\\.').test('.')
true
> new RegExp(`\.`).test('a')
true
> new RegExp(`\\.`).test('a')
false
```

Reply:
Eugh, that's annoying. Our docs are wrong in lots of places, then. We don't do anything to escape things.